### ISSUE TITLE
How to obtain the file

### ISSUE THREAD

**Issue by jnu202434551015** [Purdue-M2/AI-Face-FairnessBench#4]
if args.dataset_type == 'pair':
    train_dataset = pairDataset(args.datapath + 'train_fake_spe.csv', args.datapath + 'train_real.csv', data_transforms['train'])

May I ask how to obtain two csv files?

**Comment by Purdue-M2** [comment]
You can get those two files by splitting the train.csv into fake and real based on the Target label. The specific refers to the generation/manipulation method, Real: 0, deepfake videos: 1, GAN: 2, DM: 3.

**Comment by jnu202434551015** [comment]
Thank you very much for your reply.

May I ask about the "Predicted" in the column names (Image Path, Predicted Gender, Predicted Age, Skin Tone, Target, Skin Tone Group, Intersection) in the provided csv file? Is it really worth it?

Why do some feature rows predict Gender=1 (indicating Male) while Intersection=4 (indicating Female, Light), resulting in inconsistent gender labels?

train.csv file, for example, in the first row (in addition to the column name) is AI_Face_imagesV2/DMs/DCFACe/7856/6.jpg,1,3,6,1,1,4

**Comment by Purdue-M2** [comment]
That's the typo in the README. I have updated it. 
Intersection column meaning:
  0: Gender = Female, Skin Tone Group = Light
  1: Gender = Female, Skin Tone Group = Medium
  2: Gender = Female, Skin Tone Group = Dark
  3: Gender = Male, Skin Tone Group = Light
  4: Gender = Male, Skin Tone Group = Medium
  5: Gender = Male, Skin Tone Group = Dark

**Comment by jnu202434551015** [comment]
Thank you very much for your reply.

In the README, are there any errors in the description of the Age column (Age label: 0-Child, 1-Youth, 3-Adult, 4-Middle-aged, 5-Senior)? Should it be 0-Child, 1-Youth, 2-Adult, 3-Middle-aged, 4-Senior?

**Comment by zdaiot** [comment]
> Thank you very much for your reply.
> 
> May I ask about the "Predicted" in the column names (Image Path, Predicted Gender, Predicted Age, Skin Tone, Target, Skin Tone Group, Intersection) in the provided csv file? Is it really worth it?
> 
> Why do some feature rows predict Gender=1 (indicating Male) while Intersection=4 (indicating Female, Light), resulting in inconsistent gender labels?
> 
> train.csv file, for example, in the first row (in addition to the column name) is AI_Face_imagesV2/DMs/DCFACe/7856/6.jpg,1,3,6,1,1,4

@Purdue-M2 However, in the file training/detectors/fair_df_detector.py, there is cls_num_list=[37276, 718535, 58543, 64593, 67552, 674406, 35115, 33384], which seems to imply that there are 8 categories for Intersection ?

**Comment by misscc320** [comment]
> You can get those two files by splitting the train.csv into fake and real based on the Target label. The specific refers to the generation/manipulation method, Real: 0, deepfake videos: 1, GAN: 2, DM: 3.

Hi, I am currently working on your project and would like to know how I can obtain the train.csv and test.csv files required for training and testing. Could you please provide guidance on where to find or how to generate these files? Any help or instructions on this would be greatly appreciated!
